File: src/sentry/api/serializers/models/platformexternalissue.py
from sentry.api.serializers import Serializer, register
from sentry.models import PlatformExternalIssue


@register(PlatformExternalIssue)
class PlatformExternalIssueSerializer(Serializer):
    def serialize(self, obj, attrs, user):
        return {
            "id": f"{obj.id}",
            "issueId": f"{obj.group_id}",
            "serviceType": obj.service_type,
            "displayName": obj.display_name,
            "webUrl": obj.web_url,
        }

File: packages/sentry/fixtures/js-stubs/platformExternalIssue.js
export function PlatformExternalIssue(params = {}) {
  return {
    id: '1',
    groupId: '1',
    serviceType: 'foo',
    displayName: 'project#1',
    webUrl: 'https://example.com/1',
    ...params,
  };
}
